src/API-calls.js

```diff
@@ -48,4 +48,9 @@ export const runNameQuery = (queryString) => {
 export const fetchContactDets = (id) => {
   return fetch(`${baseURL}/contacts/${id}`)
     .then(checkResponse)
+}
+
+export const fetchIncidentContacts = (id) => {
+  return fetch(`${baseURL}/incidents/${id}/contacts`)
+    .then(checkResponse)
```

Comment:
Let's remember to ask Backend to move this endpoint up the priority list. We want to be able to build charts with this data over the weekend